variants/m5stack_station/pins_arduino.h

```diff
@@ -7,7 +7,7 @@
 #define NUM_DIGITAL_PINS        20
 #define NUM_ANALOG_INPUTS       16
 
-#define analogInputToDigitalPin(p) (((p) < 20) ? (esp32_adc2gpio[(p)]) : -1)
+#define analogInputToDigitalPin(p)  (((p) < 20) ? (analogChannelToDigitalPin(p)) : -1)
```

Comment:
Can you please remove the added space after `analogInputToDigitalPin(p)` 😄

Reply:
ok, done.